File: lua/1158.lua
local n = io.read("*n")

for i = 1, n do
    local x, y = io.read("*n", "*n")
    local sum, count = 0, 0

    while count < y do
        if x % 2 ~= 0 then
            count = count + 1
            sum = sum + x
        end

        x = x + 1
    end

    print(sum)
end

File: go/1158.go
package main

import (
	"fmt"
)

func main() {
  var n, x, y, sum, count int

  fmt.Scanf("%d", &n)

  for i := 0; i < n; i++ {
    fmt.Scanf("%d %d", &x, &y)

    sum = 0
    count = 0

    for {
      if x % 2 != 0 {
        count += 1
        sum += x
      }

      x += 1

      if count == y {
        break
      }
    }

    fmt.Println(sum)
  }

}
